Add License & Update Readme (#1)
magento/magento-cloud-patches: COPYING.txt

Copyright © 2013-present Magento, Inc.

> Each Magento source file included in this distribution is licensed under OSL-3.0 license

Reviewer: ```suggestion
Each Magento source file included in this distribution is licensed under OSL-3.0 license.
```

> Please see LICENSE.txt for the full text of the OSL-3.0 license or visit http://opensource.org/licenses/osl-3.0.php

Reviewer 2: ```suggestion
See the `LICENSE.txt` file for the full text of the OSL-3.0 license or visit http://opensource.org/licenses/osl-3.0.php
```

> Please see LICENSE.txt for the full text of the OSL-3.0 license or visit http://opensource.org/licenses/osl-3.0.php

Reviewer: ```suggestion
See LICENSE.txt for the full text of the OSL-3.0 license or visit the Open Source Initiative website: http://opensource.org/licenses/osl-3.0.php
```
